pages/S3_handler.jsx

```diff
@@ -0,0 +1,18 @@
+const secrets = require('../secrets.json');
+
+const credentials = {
+  accessKeyId:     secrets.ACCESS_KEY_ID,
+  secretAccessKey: secrets.SECRET_KEY_ID,
+  region: 'eu-central-1',
+};
+
+var AWS = require('aws-sdk');
```

Comment:
Put all `require` statements at the top of the file.

Reply:
Ofc!